mx.sulong/mx_sulong.py

```diff
@@ -1141,6 +1154,18 @@ def checkNoHttp(args=None):
     'main-arg' : runMainArgTestCases,
 }
 
+checkCases = {
+    'gitlog' : logCheck,
+    'mdl' : mdlCheck,
+    'ecj' : compileWithEcjStrict,
+    'checkstyle' : checkStyle,
+    'findbugs' : findBugs,
+    'canonicalizeprojects' : canonicalizeprojects,
+    'httpcheck' : checkNoHttp,
+    'checkoverlap' : checkoverlap,
+    'clangformatcheck' : clangformatcheck
```

Comment:
Would it be also possible to add `pylint` and `eclipseformat` in this, or another PR?